chain/chain/src/types.rs

```diff
@@ -236,6 +236,20 @@ pub struct ChainGenesis {
     pub protocol_version: ProtocolVersion,
 }
 
+pub struct ChainConfig {
+    /// Whether we should save `TrieChanges` on disk or not.
+    pub save_trie_changes: bool,
+    /// Number of threads to execute auxiliary background work.
+    /// Currently used for flat storage background creation.
+    pub background_work_threads: usize,
+}
+
+impl Default for ChainConfig {
+    fn default() -> Self {
+        Self { save_trie_changes: true, background_work_threads: 1 }
+    }
+}
```

Comment:
nit: Is this really default? I believe we only use this in tests, so wouldn't a test constructor `fn test() -> Self` be more accurate? 
Especially since the default values for this struct are defined in `impl Default for StoreConfig`.